docs/pages/ds-Hdmi-In_L3_Low-Level_TestSpecification.md

```diff
@@ -0,0 +1,63 @@
+# Device Settings HdmiIn L3 Low Level Test Specification and Procedure Documentation
+
+## Table of Contents
+
+- [Overview](#overview)
+- [Acronyms, Terms and Abbreviations](#acronyms-terms-and-abbreviations)
+- [References](#references)
+- [Level 3 Test cases High Level Overview](#level-3-test-cases-high-level-overview)
+
+## Overview
+
+This document describes the L3 Low Level Test Specification and Procedure Documentation for the Device Settings HdmiIn module.
+
+### Acronyms, Terms and Abbreviations
+
+- `HAL` \- Hardware Abstraction Layer, may include some common components
+- `UT`  \- Unit Test(s)
+- `OEM` \- Original Equipment Manufacture
+- `SoC` \- System on a Chip
+- `HDMI`\- High-Definition Multimedia Interface
+- `HDR` \- High Dynamic Range
+- `HLG` \- Hybrid Log-Gamma
+- `SDR` \- Standard Dynamic Range
+
+### References
+
+- `High Level Test Specification` - [dsHdmiIn High Level TestSpec](ds-hdmi-in_L3_High-Level_TestSpecification.md)
+- `Interface header` - [dsHdmiIn HAL header](https://github.com/rdkcentral/rdk-halif-device_settings/blob/main/include/dsHdmiIn.h)
```

Comment:
if you use main this document will move because your pointing at the tip of the latest release.

You should choose a specific version and use that URL. But state in the title `dsHdmiIn HAL header 1.x.x` is used in this definition. or something.

Reply:
done